### ISSUE TITLE
Quest: Cyclonian

### ISSUE THREAD

**Issue by underlord-oberon** [plantain-gnocchis/wow-voiceover-tbc-wotlk#54]
Add-on version: AI_VoiceOver_v1.4.5
Data version: AI_VoiceOverData_VanillaExtra_v1.4

Alterac Mountains / Chillwind Point / NPC: Bath'rah the Windwatcher / Quest: Cyclonian
Link: https://www.wowhead.com/wotlk/quest=1712/cyclonian

Description: playback ok.

Progress: no playback.

Completion: the voice-over suddenly silences during playback.

**Comment by carevass** [comment]
This should be fixed in AI_VoiceOverData_VanillaExtra_v1.5.
